### ISSUE TITLE
when using the testcafe-hammerhead proxy, open layer canvas not able to load images.

### ISSUE THREAD

**Issue by deane187** [DevExpress/testcafe#1398]
### Are you requesting a feature or reporting a bug?
Bug/issue

### What is the current behavior?
When zooming in to an image or searching for a user handle to load an image on the canvas, blank spaces are loaded. Canvas does not seem to render correctly through proxy.

### What is the expected behavior?
please see the following  [URL](http://photo-mosaic-service.bluerobot.com/7683410f-0803-11e7-8bd8-42010af003a4/index.html). Enter user handle "lucimaya295" and watch the canvas zoom in to the users image. Zoom-in and zoom-out should be seamless and smooth.

### How would you reproduce the current behavior (if this is a bug)?
Run test code  through testcafe terminal using local browser or using browserstack. Canva does not load. Only once the user handle is entered can you see some parts of the canvas.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: [http://photo-mosaic-service.bluerobot.com/7683410f-0803-11e7-8bd8-42010af003a4/index.html](http://photo-mosaic-service.bluerobot.com/7683410f-0803-11e7-8bd8-42010af003a4/index.html).

Test code

```js
import {Selector} from 'testcafe';

fixture`IG`.page`http://photo-mosaic-service.bluerobot.com/a19937b2-0803-11e7-8bd8-42010af003a4/index.html`;

test('Emulate user actions on IG mosaic and perform verifications', async t => {
  const zoomButtonsExist = Selector('div > div > div:nth-of-type(2) > div');
  const textBoxExist = Selector('.form-control');
  const findMeButtonsExist = Selector('.btn-info');

  await t
    .setNativeDialogHandler(() => true)
    .wait(3000)
    .expect(zoomButtonsExist)
    .ok()
    .expect(textBoxExist)
    .ok()
    .expect(findMeButtonsExist)
    .ok()
    .wait(3000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.ol-zoom-in')
    .wait(2000)
    .click('.form-control')
    .typeText('.form-control', `aleashalove`)
    .click('.btn-info')
    .wait(7000)
    .click('div > div > canvas')
    .wait(7000)
    .click('.modal')
    .wait(7000)
    .click('.form-control')
    .pressKey('ctrl+a')
    .pressKey('delete')
    .typeText('.form-control', `lucimaya295`)
    .click('.btn-info')
    .wait(7000)
    .click('div > div > canvas')
    .wait(7000)
    .click('.modal')
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(2000)
    .click('.ol-zoom-out')
    .wait(5000);
});
```

### Specify your

* operating system: macOS Sierra ver:10.12
* testcafe version: 0.14.0
* node.js version: 6.10.2

**Comment by AlexanderMoskovkin** [comment]
Hi @deane187,

I've reproduced the problem. It relates to our [proxy](https://github.com/DevExpress/testcafe-hammerhead). I've created an [issue](https://github.com/DevExpress/testcafe-hammerhead/issues/1087) there.

**Comment by deane187** [comment]
Hi @AlexanderMoskovkin 

Thank you !

**Comment by LavrovArtem** [comment]
Hi @deane187

We know about this issue, but unfortunately it is difficult to solve and it leads to a decrease in performance. 
Tell me please, does this issue lead to a fail in the test?

**Comment by deane187** [comment]
Hi @lavrovArtem

The tests pass or fail depending on what the proxy decides to load. When it does fail we have an issue with screenshots not showing correctly because the canvas hasn't loaded correctly.

**Comment by AlexanderMoskovkin** [comment]
We've fixed the issue. The fix will be included into `v0.15.0`, and it's already available in the latest version with the `alpha` tag (`v0.15.0-alpha4`)
